.gitignore

```diff
@@ -4,3 +4,4 @@ node_modules
 /app/dist
 /dist
 /dist-js
+*.DS_Store
```

Comment:
Nitpic, but this should be in your own global gitignore, not here.

https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Reply:
Ah, thank you! Great point.